src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSync.scala

```diff
@@ -566,6 +569,11 @@ class RegularSync(
   private def notDownloading(): Boolean =
     headersQueue.isEmpty && waitingForActor.isEmpty && !resolvingBranches
 
+  private def publishEvents(newBlocks: Seq[Block]): Unit = {
```

Comment:
Don't we need this in `handleBlocks` as well?

Reply:
true, we do